packages/trace-viewer/src/ui/shared.css

```diff
@@ -0,0 +1,66 @@
+/*
+  Copyright (c) Microsoft Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+.drop-target {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex: auto;
+    flex-direction: column;
+    background-color: var(--vscode-editor-background);
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 100;
+    line-height: 24px;
+  }
+  
+  body .drop-target {
+    background: rgba(255, 255, 255, 0.8);
+  }
+  
+  body.dark-mode .drop-target {
+    background: rgba(0, 0, 0, 0.8);
+  }
+  
+  .drop-target .title {
```

Comment:
I'd rather copy the necessary styles and not abuse `.drop-target` for the disconnected message. There is already `.ui-mode .disconnected` style, but I don't see where it is used.